client/layout/guided-tours/site-title-tour.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { translate } from 'i18n-calypso';
+import { overEvery as and } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	makeTour,
+	Tour,
+	Step,
+	Next,
+	Quit,
+	Continue,
+	Link,
+	ButtonRow,
+} from 'layout/guided-tours/config-elements';
+import {
+	selectedSiteHasDefaultSiteTitle,
+	userIsOlderThan,
+	isEnabled,
+	userCanEditSettingsOfSelectedSite,
+	isAbTestInVariant,
+} from 'state/ui/guided-tours/contexts';
+import Gridicon from 'components/gridicon';
+
+const TWO_DAYS_IN_MILLISECONDS = 2 * 1000 * 3600 * 24;
+
+export const SiteTitleTour = makeTour(
+	<Tour
+		name="siteTitle"
+		version="20161010"
+		path=""
```

Comment:
I changed it to /stats, as recommended. We experimented with empty path and having `inSection` in `when`, but this seems more simple.